build/embed_darwin_arm64.go

```diff
@@ -0,0 +1,8 @@
+package build
```

Comment:
This can be in a follow-up since it might affect the directory structure of the embed.FS, but do we need the subdirectories to all be checked in (vs a single `placeholder` file in `build/`?

Reply:
I think the layout is required in order to handle the cross-compilation wrinkle.  In particular darwin, but theoretically building on native linux might apply too.  We don't want to ingest the payloads from the "other" architecture(s) into the binary for a given architecture, which requires being explicit in the glob to filter out.